modular_coyote/eris/code/wielding.dm

```diff
@@ -21,6 +21,8 @@
 	wielded = FALSE
 	if(force_unwielded)
 		force = force_unwielded
+	if(wielded_mult)
+		force = (force / wielded_mult)
 	else
 		force = (force / 1.15)
```

Comment:
I know its not part of your PR, but while you're at it, would you please turn this 1.15 into a define? Could go in the file with the other weapon damage defines.